test/integration/puppetlabs/trapperkeeper/services/scheduler/scheduler_service_test.clj

```diff
@@ -7,53 +7,64 @@
             [overtone.at-at :as at-at]))
 
 (deftest ^:integration test-interspaced
-  (with-app-with-empty-config app [scheduler-service]
-    (testing "interspaced"
-      (let [service (tk/get-service app :SchedulerService)
-            num-runs 3 ; let it run a few times, but not too many
-            interval 300
-            p (promise)
-            counter (atom 0)
-            delays (atom [])
-            last-completion-time (atom nil)
-            job (fn []
-                  (when @last-completion-time
-                    (let [delay (- (System/currentTimeMillis) @last-completion-time)]
-                      (swap! delays conj delay)))
-                  (swap! counter inc)
-
-                  ; Make this job take a while so we can measure the duration
-                  ; between invocations and ensure that the next invocation is
-                  ; not scheduled until this one completes.
-                  (Thread/sleep 100)
+  (let [configurations [{}
+                        {:group-id :some-group-identifier}]]
+    (doseq [{:keys [group-id]} configurations]
```

Comment:
I prefer to not use `doseq` in situations like this because it makes it difficult to debug test failures as it's usually not clear which iteration(s) a given assertion failed on.  Duplicating the needed parts of the test makes the code WETter but much easier to debug when tests fail.